source/deployment-options/wazuh-from-sources/wazuh-agent/index.rst

```diff
@@ -103,6 +103,11 @@ The Wazuh agent is a single and lightweight monitoring software. It is a multi-p
 
                         # pacman --noconfirm -Syu curl gcc make sudo wget expect gnupg perl-base perl fakeroot python brotli automake autoconf libtool gawk libsigsegv nodejs base-devel inetutils cmake
 
+                .. group-tab:: APK
+
+                    .. code-block:: console
+
+                        # apk --update add git less openssh openssl openssl-dev libgcc bash make gcc g++ cmake curl automake linux-headers autoconf libtool procps python3 gettext-dev
```

Comment:
Several comments here:
- Is `git` needed? Following the documentation, the source code is downloaded by `curl`
- Why is `less` needed?
- Why is `openssh` needed?
- Why is `procps` needed? 
- Why is `gettext-dev` needed?

Let's review all these deps to avoid installing unneeded packages.

Reply:
`git`, `less`, `openssh` aren't necessary to install Wazuh, however, `procps` (is used in `execd` module) and `gettext-dev` (has some headers necessaries to build in Alpine) are necessary to working fine